flow/project.py

```diff
@@ -854,6 +897,11 @@ def add_operation(self, name, operation, directives=None):
         if directives is not None:
             self.operation_directives[name] = directives
 
+    def _add_aggregate(self, aggregate):
+        """Add :py:class:`_MakeAggregate` object associated to the FlowGroup"""
+        if self.aggregate is None:
+            self.aggregate = aggregate
+
```

Comment:
Should we just do this at the creation of the `FlowGroup`?

Reply:
This logic is to support multiple aggregations associated with an operation (if it also there in some group).
This is not necessary for now, but I think this is the right way to do it)